naas/models/email_notification.py

```diff
@@ -0,0 +1,133 @@
+import datetime
+
+from naas.models import (
+    Links, EmailNotificationStatuses, EmailNotificationDeliveries
+)
+from naas.requests import EmailNotificationStatuses as notification_status
+
```

Comment:
Error here as there is no `EmailNotificationStatuses` or `EmailNotificationDeliveries` model.